meta-mender-demo/recipes-core/psplash/psplash_git.bbappend

```diff
@@ -4,4 +4,6 @@ SRC_URI_append_mender-client-install = " \
 	file://mender.io.png \
 "
 
-SPLASH_IMAGES_mender-client-install = "file://mender.io.png;outsuffix=default"
+# Explicitly set to all arch/machines, otherwise breaks on raspberrypi builds
+# with error "Nothing RPROVIDES 'psplash-raspberrypi'"
+SPLASH_IMAGES_mender-client-install_all = "file://mender.io.png;outsuffix=default"
```

Comment:
To be honest I'm not 100% sure myself why this is needed. But can you try this instead?

```
SPLASH_IMAGES_append_mender-client-install = " file://mender.io.png;outsuffix=default"
```

(beware of the leading space inside the quotes!)

Not sure if it accomplishes the same thing as what you did, but it is more idiomatic.

Reply:
This works, too. I'll rework.

Reply:
Amended in [this commit](https://github.com/mendersoftware/meta-mender/pull/1097/commits/4fe19a1e02f951a8b376779c122fd4eeb7a9bfd4).

Reply:
@kacf Actually, it fixed raspberry builds but broke qemu ones. See this [build log](https://gitlab.com/Northern.tech/Mender/mender-qa/-/jobs/760702496), error follows:

```
NOTE: recipe psplash-0.1+gitAUTOINC+0a902f7cd8-r15: task do_populate_sysroot: Succeeded
ERROR: psplash-0.1+gitAUTOINC+0a902f7cd8-r15 do_package: QA Issue: psplash-default is listed in PACKAGES multiple times, this leads to packaging errors. [packages-list]
ERROR: psplash-0.1+gitAUTOINC+0a902f7cd8-r15 do_package: Fatal QA errors found, failing task.
NOTE: recipe psplash-0.1+gitAUTOINC+0a902f7cd8-r15: task do_package: Failed
ERROR: Logfile of failure stored in: /builds/Northern.tech/Mender/build-qemux86-64-uefi-grub/tmp/work/core2-64-poky-linux/psplash/0.1+gitAUTOINC+0a902f7cd8-r15/temp/log.do_package.8671
ERROR: Task (/builds/Northern.tech/Mender/meta/recipes-core/psplash/psplash_git.bb:do_package) failed with exit code '1'
```

Reply:
@lluiscampos: Ok, maybe just stick to the original then. Although I cannot see the exact issue, it probably has to do with the fact that `_append` causes multiple entries to end up in the list. It's only for the demo, so it's not *that* important to fix it in an idiomatic way.

Reply:
Reverted to the original workaround. If the pipeline passes I'll merge.